app/views/content/life-as-a-teacher/teaching-as-a-career/change-careers/_article.html.erb

```diff
@@ -0,0 +1,111 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+    <p>Teachers come from lots of different backgrounds and have often worked in previous careers before becoming teachers. The skills and experiences you bring from previous careers are invaluable in the classroom. From communication and public speaking, to problem solving and people management, your students will learn more from you than you might think.</p>
+    <p>So if you're interested in changing careers to become a teacher, there's lots of support available to help you find your route into teaching. </p>
+  </section>
+  <section class="col col-720">
+    <h2 class="heading--box-blue">Benefits of changing your career to teaching</h2>
+    <p>Transfering your existing skills to teaching can be incredibly rewarding. You'll be in demand as someone who has real-world experience of working in your career.</p>
+    <p>As a teacher, you can watch pupils progress as they learn, making an impact from your very first lesson.</p>
+  </section>
+  <section class="col col-720">
+    <%= render Content::QuoteComponent.new(
+    text: "I tried many things until I realised that working with young people and helping them develop skills truly inspired me.",
+    name: "Chris, career changer",
+    large: false
+  ) %>
+  </section>
+  <section class="col col-720">
+    <p>There are practical benefits too, including:
```

Comment:
You could use the partial that was created for the subject pages here - same content?

Reply:
The content is slightly different - we removed some of the links from the bullet points after a UCD review. Feedback was that it was a bit overwhelming to have so many links on the career changers page.